docs/content/preview/explore/ysql-language-features/pg-extensions/install-extensions.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Install PostgreSQL extensions
+headerTitle: Install PostgreSQL extensions
+linkTitle: Install extensions
+description: Install PostgreSQL extensions for use with YugabyteDB
+headcontent: Add extensions to your YugabyteDB installation
+menu:
+  preview:
+    identifier: install-extensions
+    parent: pg-extensions
+    weight: 10
+type: docs
+---
+
+If an extension is not pre-bundled, you need to install it manually before you can enable it using the [CREATE EXTENSION](../../../../api/ysql/the-sql-language/statements/ddl_create_extension/) statement. You can install only extensions that are supported by YugabyteDB.
+
```

Comment:
Thought that linking to the list of supported extensions here will be helpful if a user directly lands on this page.